### PR TITLE
fixed OwnerRole in underwritingConditions

### PR DESCRIPTION
**Pull request by yornstei**
OwnerRole isn't a string rather an EntityReference.

### CODE REVIEW

**Comment by yornstei** [comment]
Pleasure. 

Regarding the PR, how can I populate the `<OwnerRole>` when i create the loan using the developer connect endpoint? If the user isn't a valid user I get an error 'A valid role id must be provided in entityId under 'ownerRole' attribute'.

**Comment by TylerBrinkley** [comment]
You could try any of the users defined in the developer connect endpoint such as the `underwriter` user.

**Comment by yornstei** [comment]
I tried it (`<var addedCondition = new UnderwritingCondition { Title = "ABC", Source = ConditionSource.RecordersOffice, ForAllApplications = true , OwnerRole = new EntityReference("Underwriter", EntityType.User)};>`) though the response is `<{"summary":"Bad Request","details":"Condition 'ABC': Invalid 'entityId' - A valid role id must be provided in entityId under 'ownerRole' attribute."}>` . As the message says, it need to be a real userId.

**Comment by TylerBrinkley** [comment]
Wouldn't the OwnerRole role id be a numeric value such as "6" for Underwriter in the developer connect encompass instance?

**Comment by yornstei** [comment]
if you mean changing it to `var addedCondition = new UnderwritingCondition { Title = "ABC", Source = ConditionSource.RecordersOffice, ForAllApplications = true , OwnerRole = new EntityReference("6", EntityType.User)};` , that didn't either work. Different error though : `{"summary":"Bad Request","details":"Condition 'ABC': Invalid Entity type for Owner."}`

**Comment by TylerBrinkley** [comment]
I think the entity type would be Role.

**Comment by yornstei** [comment]
@TylerBrinkley you're correct. I updated the test.

File: src/EncompassRest.Tests/LoanConditionsTests.cs [hunk 1]
```diff
@@ -21,10 +21,11 @@ public async Task LoanConditions_GetUnderwritingConditions()
                 var conditions = await underwritingConditions.GetConditionsAsync();
                 Assert.IsNotNull(conditions);
                 Assert.AreEqual(0, conditions.Count);
-                var addedCondition = new UnderwritingCondition { Title = "ABC", Source = ConditionSource.RecordersOffice, ForAllApplications = true };
+                var addedCondition = new UnderwritingCondition { Title = "ABC", Source = ConditionSource.RecordersOffice, ForAllApplications = true, OwnerRole = new EntityReference("6", EntityType.Role) };
                 await underwritingConditions.CreateConditionsAsync(new[] { addedCondition });
                 var conditionId = addedCondition.Id;
                 Assert.IsFalse(string.IsNullOrEmpty(conditionId));
+                Assert.AreEqual(addedCondition.OwnerRole.EntityId, "6");
```
Comment (TylerBrinkley) [inline src/EncompassRest.Tests/LoanConditionsTests.cs]: expected value of "6" should be the first argument, the actual value of `addedCondition.OwnerRole.EntityId` should be the second argument.

**Comment by TylerBrinkley** [comment]
Looks good to me.

**Pull request merged**